File: Python/1. Elementos Basicos/Ejercicio #2.py
#2. Determinar la solucion logica de la siguiente operacion:
print("Operacion: ((3+5*8)<3 and ((-6/3 * 4) + 2 < 2)) or (a>b)")

a = float(input("Ingrese a: "))
b = float(input("Ingrese b: "))

RES = ((3+5*8)<3 and ((((-6/3)*4) + 2) < 2)) or (a>b)
print("El resultado de la operacion es: ",RES)

File: 1. C++/2. Expresiones/Ejercicio #2.cpp
//2. Escribe la siguiente exprecion matematica como exprecion en C++

#include<iostream>

using namespace std;

int main()
{
    int a = 0, b = 0, c = 0, d = 0;
    float res = 0;

    cout<<"Digite el valor de 'a': ";cin>>a;
    cout<<"Digite el valor de 'b': ";cin>>b;
    cout<<"Digite el valor de 'c': ";cin>>c;
    cout<<"Digite el valor de 'd': ";cin>>d;

    res = (a+b)/(c+d);

    cout<<"\nEl resultado de la operacion ("<<a<<"+"<<b<<")/("<<c<<"+"<<d<<") = "<<res<<endl;
    return 0;
}